test/winCamera.test

```diff
@@ -0,0 +1,10 @@
+<launch>
+  <node pkg="win_camera" type="win_camera_node" name="win_camera_node">
+    <param name="frame_id" value="WinCamera1" />
+    <param name="image_width" value="1280" />
+    <param name="image_height" value="720" />
+    <param name="camera_info_url" value="file://$(find win_camera)\test\sample.yaml" />
```

Comment:
not sure if this format works, but this is the way how Python ppl uses the file protocol: `file:///$(find win_camera)/test/sample.yaml`, assuming `$(find win_camara)` also returning forward-slash back.

Reply:
$(find win_camara) returns '\' in the path at least by the time it is resolved into the node process.
the problem with file:/// (3 x '/') is that only the "file://" is stripped off and one '/' is still used in the file to open. i.e the code tries to open "/e:\ros\sample.yaml" instead of "e:\ros\sample.yaml". I guess this is done to enforce full paths only in linux. However this does not work for windows.